### ISSUE TITLE
No focus when invoked from quickstart on XP/2003 64-bit

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#57]
```
What steps will reproduce the problem?

1. Drag and drop $CYGWIN_HOME/bin/mintty.exe on the left side of the
Windows task bar to create a shortcut to it there.
2. If desired, right click on the created shortcut, select the "Properties"
pop-up menu item and change the "Target" to include a "-" option (it
doesn't make a difference for this bug, but users will probably want this
in their shortcut).
3. Select that shortcut to launch MinTTY and Cygwin.


What is the expected output? What do you see instead?

The expected behavior is for the MinTTY window to come up above any other
window that was previously active and to be ready to accept Cygwin commands
immediately. Instead, what happens is that the MinTTY window comes up in
front of other windows, and its title bar indicates it is active, but it
does not accept keystrokes. It is necessary to click on the MinTTY window
before entering commands.


What version of the product are you using? On what operating system?

MinTTY 0.3.5, Cygwin 1.5.25, Windows Server 2003.


Please provide any additional information below.

The same problem does not occur when launching MinTTY directly from
$CYGWIN_HOME/bin/mintty.exe. It does occur when launching MinTTY from a
shortcut that is not on the Windows task bar, but in that case the MinTTY
window shows up behind other windows.
```

Original issue reported on code.google.com by `adxvz...@gmail.com` on 28 Feb 2009 at 12:04

**Comment by mintty** [comment]
If someone still faces this problem, and would lend me an XP 64bit VM with everything installed to reproduce it, I could have a look into it.
Otherwise, and since XP is out of support, the issue will eventually get closed again.

**Comment by mintty** [comment]
I could in fact test this on Windows XP 64 but did not reproduce the described behaviour, neither with mintty 0.4.0 nor with the current version (running current cygwin). Thus closing.

**Comment by mintty** [comment]
Leaving in the patch though that tries to enforce the focus on the new window. Released in 2.2.1.
